cmd/mr_create.go

```diff
@@ -54,7 +54,8 @@ func init() {
 	mrCreateCmd.Flags().BoolP("squash", "s", false, "squash commits when merging")
 	mrCreateCmd.Flags().Bool("allow-collaboration", false, "allow commits from other members")
 	mrCreateCmd.Flags().String("milestone", "", "set milestone by milestone title or ID")
-	mrCreateCmd.Flags().StringP("file", "F", "", "use the given file as the Description")
+	mrCreateCmd.Flags().StringP("file", "F", "", "use the given file as the Title and Description")
+	mrCreateCmd.Flags().StringP("file-open-editor", "f", "", "use the given file as the Title and Description and open the editor")
```

Comment:
What about `--file-edit` instead? Seems simpler and enough meaningful at the same time. What do you think?

Reply:
Fixed in new version :)